bot/src/config/validation-examples.ts

```
@@ -0,0 +1,180 @@
+    'DISCORD_BOT_TOKEN',
+    'MAP_API_KEY',
+  ];
+
+  const missingVars = requiredEnvVars.filter(varName => !process.env[varName]);
+  
+  if (missingVars.length > 0) {
+    console.warn(`Missing required environment variables: ${missingVars.join(', ')}`);
+  }
+
+  // Validate numeric constraints
+  const port = parseInt(process.env.PORT || '3000');
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)